contents/newsletter/this-is-why-youre-not-shipping.md

```diff
@@ -0,0 +1,211 @@
+
+![pepe silva](https://res.cloudinary.com/dmukukwp6/image/upload/pepe_silva_meme_a11b85f5ba.jpg)
+
+### What we do
+
+This is a hard problem to solve from the bottom up: it has to come from the top. At PostHog, we solve for this in a few different ways:
+
+  1.  **We operate a "no design by default" approach**. It's up to product engineers to decide if and when they require design help, and they're free to ship entire products without design input if they choose. We maintain a design system to assist them, and have [clear guidelines on how to work with design](https://posthog.com/handbook/design/philosophy).
+
+  2.  **Engineers make product decisions.** Product managers at PostHog don't decide what to build, or sign-off on features before they ship. Instead, they gather deep context on users, competitors, and potential solutions, so [product engineers can make informed decisions](https://posthog.com/newsletter/product-management-is-broken).
+
+  3.  **Trust and feedback over process.** We fundamentally believe [giving people feedback is better than creating a process](https://posthog.com/handbook/values#4-trust-and-feedback-over-process). We also do things, like [testing in production](https://posthog.com/product-engineers/testing-in-production), that reduce the risk of shipping something that breaks.
+
+This is what works for us, but it goes hand-in-hand with solving another red flag...
+
+## 3. Teams and ownership are hard to change
+
+It's hard to move fast when the wrong team owns a problem.
+
+They become a bottleneck, either by blocking people who want to fix it, or by slowing things down through excessive coordination and approvals.
+
+It's even harder to move fast when ownership problems become insoluble. People learn to avoid things that other teams own, so they go unsolved and get worse.
+
+### What we do
+
+We've talked about [how small teams ship faster](https://posthog.com/newsletter/small-teams) before, but there's another important benefit: they're easy to change.
+
+Whenever a team gets too big, or it makes sense to form a new one to achieve a goal, we just do it. Typically, the process looks something like this:
+
+![team changes at posthog](https://res.cloudinary.com/dmukukwp6/image/upload/team_change_91f7a8c7dc.png)
+
+A few months ago, for example, we created one team to own both event ingestion and building [our customer data platform](https://posthog.com/docs/cdp) and [realtime destinations](https://posthog.com/docs/cdp/destinations). This made sense because the two systems needed to work well together.
+
+Recently, however, we've split them into two separate teams again: Ingestion and CDP. Why? Because the problem has moved from "how do we make this work?" to "we need to build more integrations".
+
+We also maintain a [public feature ownership doc](https://posthog.com/handbook/company/fuzzy-ownership) that anyone can propose changes to at any time through a PR. Who owns what isn't dictated solely by leadership.
```

Comment:
```suggestion
A few months ago, for example, we created one team to own both event ingestion and building [our customer data platform](/docs/cdp) and [realtime destinations](/docs/cdp/destinations). This made sense because the two systems needed to work well together.

Recently, however, we've split them into two separate teams again: Ingestion and CDP. Why? Because the problem has moved from "how do we make this work?" to "we need to build more integrations".

We also maintain a [public feature ownership doc](/handbook/company/fuzzy-ownership) that anyone can propose changes to at any time through a PR. Who owns what isn't dictated solely by leadership.
```